.github/scripts/branch_pr_issue_closer.py

```diff
@@ -21,52 +21,34 @@
 
 import re
 import sys
-import subprocess
-import shlex
 import argparse
 
 from github import Github
+from github import Auth
+
+###############
+#REGEX PATTERNS
+###############
+
+#Issue-closing Keywords are:
+#close, closes, closed
+#fix, fixes, fixed
+#resolve, resolves, resolved
+
+#Create relevant regex patterns:
+_CLOSE_KEY              = r'close[sd]?'
+_FIX_KEY                = r'fix(e[sd])?'
+_RESOLVE_KEY            = r'resolve[sd]?'
```

Comment:
Might be worth mentioning the source webpage these get pulled from in case they change or more are added later:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Reply:
Good call!  Comment with URL has been added.